lib/taxonomy/publishing_api_adapter.rb

```diff
@@ -33,6 +31,8 @@ def level_one_taxons
 
     def expanded_links_hash(content_id)
       publishing_api_with_huge_timeout.get_expanded_links(content_id, with_drafts: false).to_h
+    rescue GdsApi::HTTPNotFound
```

Comment:
Couldn't we add world taxon test data ? Like in this commit https://github.com/alphagov/whitehall/pull/4143/commits/2120c0f230f7af290b580441913e2a6883994efc

Reply:
My first thought was to do that, but the worldwide taxonomy isn't being used in the e2e tests. Also it just felt better to make the code a bit more robust and not throw unexpected errors because another taxonomy isn't available.

Reply:
Okay dokay 😃